gdal/swig/python/samples/assemblepoly.py

```diff
@@ -41,6 +41,7 @@ def doit(name):
     ds = ogr.Open(name, update=1)
 
     layer_count = ds.GetLayerCount()
+    print('{}: Layer count: {}'.format(name, layer_count))
```

Comment:
I guess this is a debug statement to remove ?

Reply:
The line above for getting layer_count was there before and was not used. 
Now that it's in a function flake8 doesn't like it, so I was either to remove it or to do something with it...
Do you prefer to remove both?

Reply:
> Do you prefer to remove both?

Yes, no reason to output that without any good reason

Reply:
done